packages/ui/primitives/react-hooks/src/useResize.ts

```diff
@@ -0,0 +1,16 @@
+//
+// Copyright 2023 DXOS.org
+//
```

Comment:
2025?

Reply:
Huh… maybe the lint rule is acting strangely?